### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by hovaesco**
Improve `README.md` and add `dbt-trino` as a source in connection to `trino-python-client`

### CODE REVIEW

File: dbt/adapters/trino/connections.py [hunk 1]
```diff
@@ -192,7 +192,8 @@ def open(cls, connection):
             http_headers=credentials.http_headers,
             session_properties=credentials.session_properties,
             auth=auth,
-            isolation_level=IsolationLevel.AUTOCOMMIT
+            isolation_level=IsolationLevel.AUTOCOMMIT,
```
Comment (findinpath) [inline dbt/adapters/trino/connections.py:195]: Pls do this change on a separate commit.

File: README.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
 > Can I connect my dbt project to two databases?
 
 (see the answered [question](https://docs.getdbt.com/faqs/connecting-to-two-dbs-not-allowed) on the dbt website).
 
-**tldr;** `dbt` stands for transformation as in `T` within `ELT`  pipelines, it doesn't move data from source to a warehouse.
+**tldr;** `dbt` stands for transformation as in `T` within `ELT` pipelines, it doesn't move data from source to a warehouse.
 
-The creators of the `dbt` tool have added however support for handling such scenarios via
-[dbt-presto](https://github.com/dbt-labs/dbt-presto) plugin.
+`dbt-trino` adapter uses [Trino](https://trino.io/) as a underlying query engine to perform query federation across disperse data sources. Trino connects to multiple and diverse data sources ([available connectors](https://trino.io/docs/current/connector.html)) via one connection and process SQL queries at scale, while dbt handles these SQL transformation queries to create tables or views.
```
[the remark this answers is not in the document]
Comment (hovaesco) [reply README.md:15]: ohh.. I was having in mind that it's one connection in dbt, I'm going to reword it

[the remark this answers is not in the document]
Comment (hovaesco) [reply README.md:15]: right, transformation defined in dbt are pushed to Trino which handles connections to sources

thanks for catching this

File: dbt/adapters/trino/connections.py (hunk 1 above)
Comment (hovaesco) [reply dbt/adapters/trino/connections.py:195]: it's a separate commit already

**Review by findinpath — APPROVED** [review]
